### ISSUE TITLE
Why using `sim_function` for positive samples, `tf.multiply` for negative ones in the loss function?

### ISSUE THREAD

**Issue by pangedeshijie** [alibaba/graph-learn#33]
pos_logit用sim_function, 为什么 neg_logit 一直用tf.multiply呢？
  pos_logit = sim_function(src_emb, pos_emb)

  src_emb_exp = tf.tile(tf.expand_dims(src_emb, axis=1),
                        [1, per_sample_neg_num, 1])
  src_emb_exp = tf.reshape(src_emb_exp, [-1, emb_dim])
  neg_logit = tf.reduce_sum(tf.multiply(src_emb_exp, neg_emb), axis=-1
true_xent = tf.nn.sigmoid_cross_entropy_with_logits(
      labels=tf.ones_like(pos_logit), logits=pos_logit)
  negative_xent = tf.nn.sigmoid_cross_entropy_with_logits(
      labels=tf.zeros_like(neg_logit), logits=neg_logit)

  loss = tf.reduce_mean(true_xent) + 1.0 * tf.reduce_mean(negative_xent)
  logit = tf.concat([pos_logit, neg_logit], axis=-1)
  label = tf.concat([tf.ones_like(pos_logit, dtype=tf.int32),
                     tf.zeros_like(neg_logit, dtype=tf.int32)], axis=-1)

**Comment by pangedeshijie** [comment]
这个问题有人给解答吗

**Comment by baoleai** [comment]
Positive and negative samples should use the same similarity function.

**Comment by pangedeshijie** [comment]
But the negative sample in your loss function only has this formula：
neg_logit = tf.reduce_sum(tf.multiply(src_emb_exp, neg_emb), axis=-1）

Is the code wrong?

**Comment by baoleai** [comment]
Yes, we will fix it asap.
